examples/cms/src/App.tsx

```diff
@@ -0,0 +1,134 @@
+import { framer, ManagedCollectionField } from "framer-plugin"
+import "./App.css"
+import { useLayoutEffect, useState } from "react"
+import { DataSource, getDataSources, syncCollection, syncExistingCollection } from "./data"
+import { FieldMapping } from "./FieldMapping"
+import { PLUGIN_KEYS, UI_DEFAULTS } from "./constants"
+import { Spinner } from "./components/Spinner"
+
+const activeCollection = await framer.getManagedCollection()
+
+const syncDataSourceId = await activeCollection.getPluginData(PLUGIN_KEYS.SYNC_REFERENCE)
+const syncSlugFieldId = await activeCollection.getPluginData(PLUGIN_KEYS.SYNC_SLUG)
+
+const result = await syncExistingCollection(activeCollection, syncDataSourceId, syncSlugFieldId)
```

Comment:
Nice that this is now just a simple function call 👏 

I see that the result is used for many things that have nothing to do with syncing an existing collection. That makes it quite hard to understand what the result actually is

Reply:
I wonder if it would be simpler if `syncExistingCollection` would do nothing except for syncing the existing collection. Then that function is something you can forget about after reading it. You don't even need to know the implementation

Reply:
The only small thing doing so, is that in case the synchronization fails, we will re-fetch the data source when going opening the management view (we do open it automatically when sync fails, but we can also just close the plugin there).

I think it's the impact may be negligible enough to just allow it.